downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -42,6 +42,57 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### Improvements                             
 
+#### Improvements to the usage of default values of record fields
+
+Now, the default value of a record is evaluated only if a value is not provided for a specific field in the mapping 
+constructor.
+
+- With these improvements, enable the usage of mutable defaults by wrapping them with `cloneReadOnly` when creating an
```

Comment:
As suggested previously, make this a separate point, and the first point.

Reply:
Updated